GameProject/Assets/OpenWorld Assets/Scripts/trdprsnCharacter.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class trdprsnCharacter : MonoBehaviour
+{
+
+    [SerializeField] float movespeed;
+    [SerializeField] float rotatespeed;
+    //private float rotX;
+    //private float rotY;
+    [SerializeField] float rotZ;
+    private Camera cam;
+
+    private void Start()
+    {
+        cam = Camera.main;
+    }
+
+    void Update()
+    {
+        Playerturn(); // turn to face the player
+        Playerrotate(); // rotate the player itself
```

Comment:
Please fix these method names, they do not match the convention of the codebase.